planet/clients/orders.py

```diff
@@ -278,7 +279,7 @@ async def download_asset(
         location: str,
         filename: str = None,
         directory: str = None,
-        overwrite: bool = True,
+        overwrite: bool = False,
```

Comment:
It seems there should be a test that would change with the change of the default value of overwrite to False. For example: a. test that runs this function with the default value in two cases: 1. the file already exists 2. the file doesn't already exist. b. same as a but the overwrite parameter is set to True.